com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageSender.cs

```diff
@@ -2,143 +2,100 @@
 using System.Collections.Generic;
 using MLAPI.Configuration;
 using MLAPI.Internal;
-using MLAPI.Logging;
 using MLAPI.Profiling;
-using MLAPI.Security;
+using MLAPI.Serialization;
 using MLAPI.Transports;
-using BitStream = MLAPI.Serialization.BitStream;
 
 namespace MLAPI.Messaging
 {
     internal static class InternalMessageSender
     {
-        internal static void Send(ulong clientId, byte messageType, Channel channel, BitStream messageStream, SecuritySendFlags flags)
+        internal static void Send(ulong clientId, byte messageType, NetworkChannel networkChannel, NetworkStream messageStream)
```

Comment:
Nit, I don't know that channel -> networkChannel adds anything and reduces signal density, but okay

Reply:
we have `enum NetworkChannel` and `class TransportChannel` types.
personally, I didn't make this change by hand, refactoring did but I'm happy either way.